test/testcmdlineparser.cpp

```diff
@@ -466,42 +466,42 @@ class TestCmdlineParser : public TestFixture {
     void nooptions() {
         REDIRECT;
         const char * const argv[] = {"cppcheck"};
-        ASSERT_EQUALS_ENUM(CmdLineParser::Result::Exit, parser->parseFromArgs(1, argv));
```

Comment:
Since this is used in so many tests, maybe there should be a template overload that deduces the size?

Reply:
Yes, using a helper which does that would be better.